apps/nextjs/src/components/user-avatar-menu.tsx

```diff
@@ -60,10 +64,27 @@ export const UserAvatarMenu = ({ children }: UserAvatarMenuProps) => {
     });
   }, [logoutUrl, openModal, router]);
 
+  const isAdmin = session.data?.user.permissions.includes("admin");
+
   return (
     // We use keepMounted so we can add event listeners to prevent navigating away without saving the board
     <Menu width={300} withArrow withinPortal keepMounted>
       <Menu.Dropdown>
+        {isAdmin && availableUpdates && availableUpdates.length > 0 && availableUpdates[0] && (
```

Comment:
If we only show them to the admin we should also only show the indicator then. So it's probably easier if we only fetch it in the user.tsx if the user is admin and then pass RouterOutputs[*] | undefined down. Then the logic for available updates does not need to change